test/e2e/loopback_test.go

```diff
@@ -308,6 +308,9 @@ var _ = ginkgo.Describe("Loopback registration [development]", func() {
 			var err error
 			managedCluster, err = managedClusters.Get(context.TODO(), clusterName, metav1.GetOptions{})
 			if err != nil {
+				if err.(*errors.StatusError).ErrStatus.Reason == metav1.StatusReasonNotFound {
```

Comment:
you can use `k8s.io/apimachinery/pkg/api/errors` to judge not found, e.g.

```
        if errors.IsNotFound(err) {
		return false, nil
	}
	if err != nil {
		return false, err
	}
```

Reply:
Done